Xamarin.Forms.Platform.Android/Renderers/EntryRenderer.cs

```diff
@@ -220,7 +220,17 @@ protected override void Dispose(bool disposing)
 		}
 
 
-		protected virtual void UpdatePlaceHolderText() => EditText.Hint = Element.Placeholder;
+		protected virtual void UpdatePlaceHolderText()
+		{
+			if (EditText.Hint != Element.Placeholder)
+			{
+				EditText.Hint = Element.Placeholder;
+				if (EditText.IsFocused)
+				{
+					EditText.ShowKeyboard();
+				}
+			}
+		}
```

Comment:
here you can use the early return pattern

Like this:

```C#
			if (EditText.Hint == Element.Placeholder)
				return;

			EditText.Hint = Element.Placeholder;
			if (EditText.IsFocused)
			{
				EditText.ShowKeyboard();
			}
```

Reply:
Yes sure. Will change it.